modules/cnf-about-ptp-fast-event-notifications-framework.adoc

```diff
@@ -6,24 +6,31 @@
-.Overview of PTP fast events
-image::218_OpenShift_PTP_events_0222.png[Overview of PTP fast events]
-
-The `cloud-event-proxy` sidecar container can access the same resources as the primary vRAN application without using any of the resources of the primary application and with no significant latency.
-
-The fast events notifications framework uses a REST API for communication and is based on the O-RAN REST API specification. The framework consists of a publisher, subscriber, and an AMQ messaging bus to handle communications between the publisher and subscriber applications. The `cloud-event-proxy` sidecar is a utility container that runs in a pod that is loosely coupled to the main DU application container on the DU node. It provides an event publishing framework that allows you to subscribe DU applications to published PTP events.
-
-DU applications run the `cloud-event-proxy` container in a sidecar pattern to subscribe to PTP events. The following workflow describes how a DU application uses PTP fast events:
-
-. *DU application requests a subscription*: The DU sends an API request to the `cloud-event-proxy` sidecar to create a PTP events subscription. The `cloud-event-proxy` sidecar creates a subscription resource.
+The framework consists of a publisher, subscriber, and an AMQ or HTTP messaging bus to handle communications between the publisher and subscriber applications.
+Applications run the `cloud-event-proxy` container in a sidecar pattern to subscribe to PTP events.
+The `cloud-event-proxy` sidecar container can access the same resources as the primary application container without using any of the resources of the primary application and with no significant latency.
 
-. *cloud-event-proxy sidecar creates the subscription*: The event resource is persisted by the `cloud-event-proxy` sidecar. The `cloud-event-proxy` sidecar container sends an acknowledgment with an ID and URL location to access the stored subscription resource. The sidecar creates an AMQ messaging listener protocol for the resource specified in the subscription.
-
-. *DU application receives the PTP event notification*: The `cloud-event-proxy` sidecar container listens to the address specified in the resource qualifier. The DU events consumer processes the message and passes it to the return URL specified in the subscription.
-
-. *cloud-event-proxy sidecar validates the PTP event and posts it to the DU application*: The `cloud-event-proxy` sidecar receives the event, unwraps the cloud events object to retrieve the data, and fetches the return URL to post the event back to the DU consumer application.
-
-. *DU application uses the PTP event*: The DU application events consumer receives and processes the PTP event.
+[NOTE]
+====
+You enable the `cloud-event-proxy` sidecar container by setting the `enableEventPublisher` field to `true` in the `ptpOperatorConfig` custom resource (CR) and specifying an Advanced Message Queuing Protocol (AMQP) or HTTP `transportHost` address.
```

Comment:
in the case HTTP Transport host it is fixed with the NODE_NAME variable as part of the service name. can we add that here 
http://ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043

Reply:
this wont change for PTP, probably we should do it as default if no transportHost is mentioned

Reply:
Going to remove this note, since it's covered in the "Configuring the PTP fast event notifications publisher" section anyway.